gytheio-messaging/gytheio-messaging-camel/pom.xml

```diff
@@ -11,7 +11,7 @@
     <artifactId>gytheio-messaging-camel</artifactId>
 
     <properties>
-        <dependency.camel.version>2.13.2</dependency.camel.version>
+        <dependency.camel.version>2.22.3</dependency.camel.version>
```

Comment:
FYI: ACS Repo will be updating to 2.23.1 (see REPO-4241)

Reply:
Similar to above, I figured Repo could exclude these dependencies and bring in their own.

Reply:
Repo provides Gytheio ! I highly recommend we're consistent whereever possible.

Reply:
Sorry, I misread the version in your comment.  Indeed we can update to that version here.